README_BUILD.md

```diff
@@ -44,6 +45,43 @@ The standard workflow when building the engine is the following:
 
 When working on a new feature or fixing a bug you start by first building the engine once as described above. You then proceed to develop your feature or fix the bug and rebuild and test changes until satisfied. When you do a rebuild you can speed things up by only building the parts that have changed.
 
+## Build examples
```

Comment:
To get to the "good stuff" a little faster, I present a few examples upfront.
Currently visible on the first page when visiting this readme.